testing/py3-oscrypto/APKBUILD

```diff
@@ -0,0 +1,37 @@
+# Contributor: Sören Tempel <soeren+alpine@soeren-tempel.net>
+# Maintainer:
+pkgname=py3-oscrypto
+_pkgname=${pkgname##py3-}
+pkgver=0.19.1
+pkgrel=0
+pkgdesc="Python crypto library backed by the OS"
+url="https://github.com/wbond/oscrypto"
+arch="noarch"
+license="MIT"
+depends="python3 py3-setuptools py3-asn1crypto"
+makedepends="python3-dev"
```

Comment:
with noarch it is overwhelmingly that python3-dev is not required, so it can be removed